### ISSUE TITLE
fails with autonumber for sequenceNumbers

### ISSUE THREAD

**Issue by deild** [raghur/mermaid-filter#41]
When use autonumber option in a sequenceDiagram Mermaid filter fails and appears to hangs indefinitely.

## Steps to reproduce

use a markdown file such as

```
~~~mermaid
sequenceDiagram
    autonumber
    Alice->>John: Hello John, how are you?
~~~
```

**Comment by raghur** [comment]
Can you check if it hangs with mmdc cli directly as well? This project depends on [mermaid cli](https://github.com/mermaidjs/mermaid.cli) & Mermaid CLI hangs when it encounters an error. Also there doesn't seem to be much dev activity around it.

If it doesn't hang with mermaid cli, please let me know in a comment below and I'll dig in. Sorry!

The only silver lining here is that the core mermaidjs folks have forked the CLI repo (no releases yet and it doesn't work yet) - but likely it would be better supported.

**Comment by deild** [comment]
The problem is solved since version 1.4.3 and the upgrade of Mermaid cli

Thank you
